### PR TITLE
Controller documentation changes.

### PR DESCRIPTION
**Pull request by jyang-cpr**

### CODE REVIEW

File: components/ps4_controller_pairing.mdx [hunk 1]
```diff
@@ -1,21 +1,44 @@
...
+
+2. If `ds4drv` is not installed, install it by running:
+
+```
+sudo apt-get install python-ds4drv
+```
+
+3. Put the controller into pairing mode by pressing and holding the SHARE and PS buttons until the controller's LED light bar flashes rapidly in white. 
+
+4. SSH into the robot and run the controller pairing script:
```
Comment (rhysfaultless-cpr) [inline components/ps4_controller_pairing.mdx]: @jyang-cpr is SSH assumed from step 1?

**Review by rhysfaultless-cpr — CHANGES_REQUESTED** [review]
Refer to the one question related to SSH.

File: components/ps4_controller_pairing.mdx (hunk 1 above)
Comment (jyang-cpr) [reply components/ps4_controller_pairing.mdx]: Yes. I added some clarification in: https://github.com/clearpathrobotics/cpr-documentation/pull/63/commits/60600d89fd51f1dc76781755ee5d26ce36919def

**Review by rhysfaultless-cpr — APPROVED** [review]
Looks good @jyang-cpr.

**Pull request merged**